test/cypress/integration/social-share.test.js

```diff
@@ -0,0 +1,56 @@
+const socialShareButton = '[data-test="socialShareButton"]'
+const socialShareLinks = '[data-test="socialShareLinks"]'
+const socialShareSnackbar = '[data-test="socialShareSnackbar"]'
+
+describe('Social share', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('Social share expands on click and collapse on click.', () => {
+    cy.get(socialShareButton).click()
+    cy.get(socialShareLinks).children().each(link => {
+      cy.wrap(link).should('be.visible')
+    })
+    cy.get(socialShareButton).click()
+    cy.get(socialShareLinks).children().each(link => {
+      cy.wrap(link).should('not.be.visible')
+    })
+  })
+
+  it('Social share expands on enter and collapse on enter.', () => {
+    cy.get(socialShareButton).focus()
+    cy.get(socialShareButton).trigger('click')
+    cy.get(socialShareLinks).children().each(link => {
+      cy.wrap(link).should('be.visible')
+    })
+    cy.get(socialShareButton).trigger('click')
+    cy.get(socialShareLinks).children().each(link => {
+      cy.wrap(link).should('not.be.visible')
+    })
+  })
+
+  it('Social share links should collapse when a link is clicked.', () => {
+    const button = cy.get(socialShareButton)
```

Comment:
Can you refactor this for consistency?
Instead of the button you used `cy.get(socialShareButton).click()`.
Instead of `button.children() you used `cy.get(socialShareLinks).children()`